tools/webpack/packages.js

```diff
@@ -31,44 +29,47 @@ const gutenbergPackages = Object.keys( dependencies )
 	)
 	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) );
 
+const exportDefaultPackages = [
+	'api-fetch',
+	'deprecated',
+	'dom-ready',
+	'redux-routine',
+	'token-list',
+	'server-side-render',
+	'shortcode',
+	'warning',
+];
```

Comment:
It would be really nice if this information was part of the individual packages, like a flag in `packages/api-fetch/package.json`. The list shoudn't be maintained in a build script.

It's certainly a task for another PR, it just caught my attention here and I wanted to make a note.

Reply:
I agree that it's a bit unfortunate that we have to list all of them. The benefit would be that we wouldn't have to maintain the same list in WordPress core. Still, it's a manual task that we will have to remember about.